stack.go

```diff
@@ -46,7 +46,7 @@ func (f Frame) line() int {
 //
 // Format accepts flags that alter the printing of some verbs, as follows:
 //
-//    %+s   path of source file relative to the compile time GOPATH
+//    %+s   package name, function name, and absolute path of source file at compile time
```

Comment:
@davecheney btw this is the accurate documentation for `%+s` if you want to pick this in